types/apollo-upload-client/tsconfig.json

```
@@ -4,7 +4,7 @@
         "lib": [
             "es6",
             "dom",
-            "esnext.asynciterable"
+            "esnext"
```

```suggestion
            "ES2018.AsyncGenerator",
            "ES2018.AsyncIterable"
```
it looks it needs only those